lib/dev_server.dart

```diff
@@ -84,6 +93,30 @@ class ButterflyDevServer {
     request.response.write(JSON.encode(result));
   }
 
+  Future<Null> _serveDevServerClientFile(HttpRequest request) async {
+    final packagesFile = new File('.packages');
+
+    if (!await packagesFile.exists()) {
+      throw new StateError('.packages file not found. Have you run pub get?');
+    }
+
+    List<String> packageInfoParts = await packagesFile
+        .openRead()
+        .transform(const Utf8Decoder())
+        .transform(const LineSplitter())
+        .map((String line) => line.split(':'))
```

Comment:
Still deciding whether to use Effective Dart or Flutter style guides. When we decide which to use, let's use the linter to enforce it. Otherwise, it will be hard to keep it consistent.